libcontainer/cgroups/fs/cpu.go

```diff
@@ -19,24 +19,19 @@ func (s *CpuGroup) Name() string {
 	return "cpu"
 }
 
-func (s *CpuGroup) Apply(path string, d *cgroupData) error {
-	// This might happen if we have no cpu cgroup mounted.
-	// Just do nothing and don't fail.
-	if path == "" {
-		return nil
-	}
```

Comment:
Presumably this code could never be reached before or something? My guess is that it probably existed for some of the older RHEL versions which had very funky cgroup setups and needed all sorts of workarounds (which are probably very outdated at this point).

Reply:
This is added by https://github.com/opencontainers/runc/pull/860/commits/950700e73c20c64deda10f944533363fda494d47 and I've checked that the current codebase do not call `Apply()` with empty path before removing this.

Reply:
I wasn't doubting you were being thorough, was just surprised by the removal. Thanks for checking!